src/Workspaces/Core/Portable/FindSymbols/Extensions.cs

```diff
@@ -19,20 +19,6 @@ internal static partial class Extensions
         public const string ComplexArrayReceiverTypeName = "[]";
         public const string ArrayReceiverTypeNameSuffix = "[]";
 
-        public static async Task<IEnumerable<SyntaxToken>> GetConstructorInitializerTokensAsync(
-            this Document document, SemanticModel model, CancellationToken cancellationToken)
-        {
-            var root = await model.SyntaxTree.GetRootAsync(cancellationToken).ConfigureAwait(false);
-
-            var syntaxFacts = document.GetLanguageService<ISyntaxFactsService>();
-            if (syntaxFacts == null)
-            {
-                return SpecializedCollections.EmptyEnumerable<SyntaxToken>();
-            }
-
-            return FindReferenceCache.GetCache(model).GetConstructorInitializerTokens(syntaxFacts, root, cancellationToken);
-        }
```

Comment:
only used by single caller (taht already had access to FindReferencesCache, so no need for this.